src/js/webrequest.js

```diff
@@ -117,13 +117,19 @@ function onBeforeRequest(details){
  * @returns {*} modified headers
  */
 function onBeforeSendHeaders(details) {
-  let frame_id = details.frameId,
+  if (badger) {
+    setTimeout(function() {
+      return badger.heuristicBlocking.heuristicBlockingAccounting(details);
```

Comment:
No need to use return, heuristicBlockingAccounting doesn't return anything and nobody will get the returned value out of a setTimeout anyway.